dataland-api-key-manager/src/main/kotlin/org/dataland/datalandapikeymanager/services/ApiKeyManager.kt

```diff
@@ -25,8 +26,12 @@ class ApiKeyManager(
 
     companion object {
         private const val milliSecondsInADay = 86400000
+        private const val defaultMaxDaysOfValidityForApiKey = 365
     }
 
+    @Value("\${dataland.max-days-of-validity-for-api-key}")
+    private var maxDaysOfValidityForApiKey = defaultMaxDaysOfValidityForApiKey
```

Comment:
Kann die Value Annotation auf private members angewendet werden? Müsste eigentlich schon gehen.

Reply:
Offensichtlich muss es wohl klappen.
Grund:   Wenn du den Api-Key-Manager aktuell startest, kannst du z.B. `daysValid=3000` machen und einen Api Key generieren.    Wenn er den value aus den application.properties nicht kriegt bzw. dort nicht findet, nimmt er als default 365.
D.h. offensichtlich nimmt er nicht den default, weil ja 3000 geht.


Aber mir ist gerade was anderes aufgefallen.
Anstatt den default oben im `companion` als `defaultMaxDaysOfValidityForApiKey` zu definieren, müsste ich auch einfach hier unten schreiben können:

`@Value("\${dataland.max-days-of-validity-for-api-key:365}")

Ist schöner.

Was meinst du?

Reply:
Stimmt. Klingt gut.

Reply:
Ich habe es gemacht wie oben beschrieben: `@Value("${dataland.max-days-of-validity-for-api-key:365}")`

Jedoch ging das nur, wenn der Value in den Konstruktor vom ApiKeyManager gegegeben wird.  Habe es daher gemacht.